### PR TITLE
Move custom bits from customizer and into repo

### PR DESCRIPTION
**Pull request by jhogue**
1. Archives, center justify the vehicles for when there are fewer than 3 at desktop layout
2. Remove the curly quote from citations… Client was not impressed :P

### CODE REVIEW

File: sass/site/primary/_posts-and-pages.scss [hunk 1]
```diff
@@ -152,12 +152,12 @@
 // "Snipe" across top corner of cards for "Sold" vehicles
 .card.type-vehicle.tag-sold,
 .tag-sold .entry-content .ngg-gallery-thumbnail {
-	
+
```
Comment (kangabell) [inline sass/site/primary/_posts-and-pages.scss:155]: yes to removing weird space!

**Comment by kangabell** [comment]
@jhogue have you tested this code merged into `master` in a real environment yet? do you think it needs to be? seems like pretty safe changes but just want to ask so we're not like "wtf happened" whenever we deploy code again

**Comment by jhogue** [comment]
I have not, but I considered that since this code was already working via the customizer that it would be very safe to merge it in. Once that is done I will remove the rules that are there from the Customizer on Production.

**Comment by kangabell** [comment]
Okay! I will double-check it when I work on the other stuff I was going to do for SVI, but if that doesn't happen soon for some reason I will just merge and deploy it.

**Pull request merged**